io.github.simple64.simple64.yaml

```diff
@@ -0,0 +1,72 @@
+app-id: io.github.simple64.simple64
+runtime: org.kde.Platform
+runtime-version: "6.3"
+sdk: org.kde.Sdk
+command: gui-wrapper.sh
+rename-icon: simple64
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --socket=pulseaudio
+  - --share=network
+  - --device=all
+  - --talk-name=org.freedesktop.ScreenSaver
+modules:
+  - shared-modules/libusb/libusb.json
+
+  - name: 7zip
+    buildsystem: simple
+    build-commands:
+      - install -D 7zz -t /app/bin
+      - ln -s /app/bin/7zz /app/bin/7za
+    sources:
+      - type: archive
+        url: https://www.7-zip.org/a/7z2107-linux-x64.tar.xz
+        sha256: 4e2499064413ea66c6b83beb994f0ea4ee87b1ec80188188db62a6ec7e74d300
+        strip-components: 0
+
+  - name: hidapi
+    buildsystem: cmake
+    sources:
+      - type: git
+        url: https://github.com/libusb/hidapi.git
+        tag: hidapi-0.12.0
+
+  - name: simple64
+    buildsystem: simple
+    build-options:
+      cflags-override: true
+      cppflags-override: true
+      cxxflags-override: true
+      ldflags-override: true
+    build-commands:
+      - ./build.sh nozip
+      - mv simple64 /app
+    sources:
+      - type: git
+        url: https://github.com/simple64/simple64.git
+        tag: v2022.08.9
+        commit: fbf2c912a7f2f8bdfd26a686bd679418a0ed3270
+        x-checker-data:
+          is-main-source: true
+          type: git
+          tag-pattern: ^(v[\d.]+)$
+
+  - name: finalize
+    buildsystem: simple
+    build-commands:
+      - install -D simple64.svg -t /app/share/icons/hicolor/scalable/apps
+      - install -D io.github.simple64.simple64.desktop -t /app/share/applications
+      - install -D io.github.simple64.simple64.metainfo.xml -t /app/share/metainfo
+      - install -D gui-wrapper.sh -t /app/bin
+    sources:
+      - type: file
+        url: https://raw.githubusercontent.com/simple64/simple64-gui/main/simple64.svg
+        sha256: ab48d530d945dab710df100b8cd21c105e7917b013f9daea213f086fb1edb9ce
```

Comment:
this is bound to fail. You should reference an explicit revision of the file so that if it ever change in the future it doesn't break the build.

Reply:
I'm just trying to rename the app